gnoland/website/views/faucet.html

```diff
@@ -13,22 +13,30 @@
         <pre id="source">
 
 This is the Gno.land (test) faucet.
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<script type="text/javascript">
+    function main() {
+        // XXX
+    };
 
-<form method="POST" action="http://gno.land:5050/">
+    {{ if .Data.captcha }}
+    function recaptchaCallback() {
+        document.getElementById("submitBtn").disabled = false;
+    };
+    {{ end }}
+</script>
+<!-- <form method="POST" action="http://gno.land:5050/"> -->
+<form method="POST" action="http://localhost:5050/">
```

Comment:
```suggestion
<form method="POST" action="{{ .Data.faucetURL }}">
```

We’ll profit from the fact that we now have a templating system to set this dynamically.